components/layout/Header.tsx

```diff
@@ -1,191 +1,212 @@
-import CloseIcon from '@mui/icons-material/Close';
-import Menu from '@mui/icons-material/Menu';
+import { ArrowForwardIos, ChevronRight, Menu } from '@mui/icons-material';
```

Comment:
Is there a diff between ArrowForwardIos and the ArrowForwardIcon that we are using in the eligibilitybutton component?

Reply:
Yes, they are different icons. 

<img width="602" alt="image" src="https://github.com/user-attachments/assets/b964899c-b9cf-4964-9e97-3dec47352bc5">
<img width="605" alt="image" src="https://github.com/user-attachments/assets/5bf3b84c-32e7-4808-8132-d16b6cd6d8be">